docs/faq.md

```diff
@@ -0,0 +1,148 @@
+
+Additionally, meshcloud GmbH has its own incident response team. The team is responsible for handling reported incidents along with informing the customers about major changes in their Information Security Management.
+
+### How is my data secured within meshStack?
+
+The cloud components used by meshStack are configured according to best practices.
+
+Hard disk encryption or container services with encrypted storage are used for different operations. Cloud storage such as AWS S3 is used to store & backup encrypted customer data and databases. The cloud components are also configured to encrypt the data in transit.
+
+- [**AWS data encryption whitepaper**](https://docs.aws.amazon.com/whitepapers/latest/introduction-aws-security/data-encryption.html)
+- [**GCP encryption at rest whitepaper**](https://cloud.google.com/docs/security/encryption/default-encryption)
+- [**GCP encryption in transit whitepaper**](https://cloud.google.com/docs/security/encryption-in-transit)
+- [**Azure encryption overview**](https://docs.microsoft.com/en-us/azure/security/fundamentals/encryption-overview)
+
+### How does meshStack securely handle my cloud platform credentials?
+
+In order to connect to the cloud platforms and execute replication steps or metering credentials to the cloud platform have to be provided to meshStack. Asymmetric encryption is used for these secrets. The service that actually stores newly entered secrets encrypts them with a public key. Only the replicator and metering services can read their according secrets with their private key to actually connect to the cloud platforms.
+
+### How does meshcloud GmbH secure the Infrastructure for my meshStack?
+
+The various cloud components used by meshStack, such as Virtual Cloud Network, Network Load Balancers, and Ingress Controller (Kubernetes) are configured according to best practices.
+
+Kubernetes nodes are secured by a shared responsibility model. meshcloud GmbH uses best-practice configurations & network policies to secure the nodes for your meshStack. Access to these Kubernetes nodes is possible through the project/role assignment of the respective cloud provider.
+
+- [AWS EKS security overview](https://docs.aws.amazon.com/eks/latest/userguide/security.html)
+- [GCP GKE security overview](https://cloud.google.com/kubernetes-engine/docs/concepts/security-overview)
+- [Azure AKS security overview](https://docs.microsoft.com/en-us/azure/aks/concepts-security)
+
+Technical safeguards are implemented such as detecting and responding to Distributed Denial-of-Service (DDoS) attacks, data encryption, etc by leveraging cloud provider services.
+
+A dedicated Virtual Cloud Network is provided for the meshStack. This network is completely encapsulated from other meshStack and their dedicated customer networks. The network configuration also includes the usage of Security Groups to strictly configure communication in the network.
```

Comment:
I think this is wrong for our Kubernetes meshStack as we only have 1 VPC and encapsulate using network policies. Or am I wrong @henryde ?